crates/astria-sequencer/src/fees/storage/snapshots/astria_sequencer__fees__storage__keys__tests__keys_should_not_change.snap

```diff
@@ -0,0 +1,19 @@
+---
+source: crates/astria-sequencer/src/fees/storage/keys.rs
+assertion_line: 48
+expression: Helper
```

Comment:
This `Helper` gives us one snapshot file, but at the cost of no longer having a pinpoint to which exact line failed.

If we want to have snapshot files, then we should buy in completely and actually have 14 files and the exact expressions that failed.

Alternativeliy, we can store the expected results inline:
```rust
insta::assert_snapshot!(the_expression, @"the_result");
```
For example:
```rust:
insta::assert_snapshot!(IBC_SUDO_CHANGE, @"fees/ibc_sudo_change");
```